packages/classic-test-app/package.json

```diff
@@ -38,7 +38,7 @@
     "ember-cli-inject-live-reload": "^2.0.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "4.0.2",
-    "ember-data": "~4.4.0",
+    "ember-data": "~4.10.0",
```

Comment:
Please rollback the change and we should be good to go with the PR.

[Test scenarios](https://github.com/mainmatter/ember-simple-auth/blob/master/packages/ember-simple-auth/config/ember-try.js) should be using new ember-data correctly now :+1: